### ISSUE TITLE
Azure Update Management - Automation account stuck at loading

### ISSUE THREAD

**Issue by gbelangeiro** [microsoft/Windows-Admin-Center-Ideas-and-Feedback#40]
Version (1.3.2105.24004)

**To Reproduce**
1. Go to my server
2. Click on update, sign in Azure account, Update Now to 
Centrally manage updates on all of your servers by using Azure Update Management
3. Choose my Azure Subcription, use my log analytics workspace, use my automantion account
4. stuck on loading for my automation account

**Expected behavior**
I'm try to add local servers in my Azure Update Management from WAC. When I arrive to Automation Account, it stuck at loading. No error message, just looping. We already have 10 local (physical and VM) servers in my Azure Update Management. 
I found that we have some UI error in the event log : msft.sme.windows-update
source : appErrorHandler
timestamp : 1631024859385
message : ajax error 400

or 

msft.sme.packetmon!main: RemoteException: The term 'pktmon' is not recognized as the name of a cmdlet

and in the dev tool of the browser (Firefox or Edge) we have this : 

{"error":{"code":"AuthenticationFailed","message":"Authentication failed. The 'Authorization' header is missing."}}

**Screenshots & Additional context**
![WAC](https://user-images.githubusercontent.com/46754734/134409091-a803f9c2-9aef-486f-a92a-3d66f6a66622.PNG)

**Comment by andersgidlund** [comment]
> Hello @gbelangeiro,
> 
> This has been fixed. We will push out an update to the extension soon.

When will this be pushed? Im having problems adding a customers server to Azure Update Management, its exactly this issue. Its been almost 3 weeks since you wrote that it would be pushed out "soon".

**Comment by RyanMesser** [comment]
@trungtran-msft  @andersgidlund 
Can confirm I'm also having this issue, first server add OK but then after that others do not and get stuck. I'm on version 1.3.2111.01001

**Comment by MarcLaf** [comment]
Just updated WAC to latest version 2110 and it appears to have lost connectivity to Azure Update Management. I'm trying to fix it but experiencing the same issue as this post - stuck at loading existing automation accounts.

**Comment by andersgidlund** [comment]
> Hello @gbelangeiro,
> 
> This has been fixed. We will push out an update to the extension soon.

How is it going with this issue?

**Comment by gbelangeiro** [comment]
@andersgidlund The problem is always there. 
@trungtran-msft Any update ???

**Comment by stevenwest99** [comment]
Still an issue today.  Automation fails with invalid request.

**Comment by jkoch22** [comment]
Receiving a 400 error when trying to load the automation accounts. It appears that the API version is not compatible with the resource provider. 

{"error":{"code":"NoRegisteredProviderFound","message":"No registered resource provider found for location 'westcentralus' and API version '**2020-03-01-preview**' for type 'automationAccounts'. The supported api-versions are '**2015-01-01-preview, 2015-10-31, 2017-05-15-preview, 2018-01-15, 2018-06-30, 2019-06-01, 2020-01-13-preview, 2021-04-01, 2021-06-22**'. The supported locations are 'japaneast, eastus2, westeurope, southafricanorth, ukwest, switzerlandnorth, brazilsoutheast, norwayeast, germanywestcentral, uaenorth, switzerlandwest, japanwest, uaecentral, australiacentral2, southindia, francesouth, norwaywest, westus3, swedencentral, southeastasia, southcentralus, northcentralus, eastasia, centralus, westus, australiacentral, australiaeast, koreacentral, eastus, westus2, brazilsouth, uksouth, westcentralus, northeurope, canadacentral, australiasoutheast, centralindia, francecentral, jioindiawest, jioindiacentral, canadaeast'."}}

**Comment by cheapscotman** [comment]
Still a problem

**Comment by MarcLaf** [comment]
Has this been fixed yet?

**Comment by trungtran-msft** [comment]
I apologize for the delay. The fix went out in November but it appears that the fix for this has been regressed. I've already made the engineers aware and will hopefully have it fixed.

**Comment by MarcLaf** [comment]
Thanks @trungtran-msft for the update.

**Comment by andersgidlund** [comment]
> I apologize for the delay. The fix went out in November but it appears that the fix for this has been regressed. I've already made the engineers aware and will hopefully have it fixed.

What does this mean exactly? Was it fixed and pushed but  then the same error arised again or was the fix missed in the November update?

**Comment by winmasterxp** [comment]
> I apologize for the delay. The fix went out in November but it appears that the fix for this has been regressed. I've already made the engineers aware and will hopefully have it fixed.

Could you provide an update please? Are we waiting on a new update for WAC to be released?

**Comment by MarcLaf** [comment]
Saw that a patch was released for WAC but no mention of this being fixed.

**Comment by winmasterxp** [comment]
> Saw that a patch was released for WAC but no mention of this being fixed.

@MarcLaf I have applied the WAC update, but this is still an issue. (I believe its isolated down to the extension its self.)

I find it disappointing that this has been an issue for over 6+ months with no fix in sight. I'm aware that I can manually link the servers to Azure Update outside of WAC, but if MS actually wants us to use this tool and, more importantly, convince others to do so as well then we need more communication and updates. (At least the long running RBAC bug that was introduced a while ago after an update was addressed in this update!)

**Comment by MarcLaf** [comment]
@winmasterxp I'm beginning to think it was caused by permission requirement changes in Azure but haven't been able to pinpoint where or what. Every so often I try looking around to verify that it's all setup properly but it's a bit of a jumble.

**Comment by RebeccaMbulaMs** [comment]
Resource to 2110.2 release blog: https://techcommunity.microsoft.com/t5/windows-admin-center-blog/windows-admin-center-version-2110-2-is-now-generally-available/ba-p/3294253

**Comment by MarcLaf** [comment]
Hi @RebeccaMbulaMs thank you for getting in touch. I'd love to be able to test it in the new release but the auto update function still isn't working and the scheduled install date is the 12th on my console. I would try to manually install it however the quest to find the install file has put me on a wild goose chase around in circles with zero luck. I keep getting redirected to either https://www.microsoft.com/en-us/download or running Windows Server on Azure. I'm starting to think I'm losing my mind. Even the direct link https://www.microsoft.com/en-us/windows-server/trial which I thought had the files doesn't. There's not even a "download" on this page.

**Comment by RebeccaMbulaMs** [comment]
Yes, just checked and confirmed the link to the .2 release is not accessible. I am really sorry about this. I have filed an bug on the updates issue and put it for triage. I will get you a working link.

**Comment by jkoch22** [comment]
@MarcLaf I've always used the [https://aka.ms/wacdownload](https://aka.ms/wacdownload) webpage. This gives me the 2110.2 download.

**Comment by RebeccaMbulaMs** [comment]
Thank you @jkoch22 , yes, this is the link for the 2110.2 release. @MarcLaf would you mind if you could try it and send me an update to see if the issue is still present?

**Comment by winmasterxp** [comment]
@RebeccaMbulaMs I'm currently on 2110.2 and my update extension is version 2.9.0. Unfortunately, it doesn't work.

**Comment by RebeccaMbulaMs** [comment]
@winmasterxp Thank you for confirming this to me. I will file this as a bug and hopefully have updates on it soon!

**Comment by MarcLaf** [comment]
@RebeccaMbulaMs I just tested and yes it's the same for me too - hangs at loading automation account.

**Comment by RebeccaMbulaMs** [comment]
@MarcLaf Thank you for that. 
Just looked into the bug from our end and it was marked as resolved. I have reopened it and we are looking into it.
Thank you for your patience!

**Comment by winmasterxp** [comment]
Hello everyone,
Yesterday on a whim, I decided to see if this was still and an issue and to my surprise, I was able to add several severs to Azure Update Management. I checked and I'm still on build 2110.2 (Version 1.3.2204.19002) & my update extension is still 2.9.0 so I'm guessing the issue was resolved on the back end.

**Comment by MarcLaf** [comment]
@winmasterxp I just tried it and although it doesn't spin anymore and I can proceed with the option to add a server to AUM (and it says successful), after refreshing the page it appears to not stick. The server doesn't actually get added to AUM and I can go through the steps again with the same results. So it looks half fixed for me.

**Comment by cheapscotman** [comment]
WAC works fine right now after update can add systems to update

**Comment by yanuoma** [comment]
Thanks for the original report, @gbelangeiro, and apologies for the long delay in our triage cycle.

We've reviewed this against the current Windows Admin Center platform and believe the underlying behavior has been addressed. The product has evolved significantly since this report was filed.

If you can still reproduce this on a current WAC build, please reply here and we'll reopen with a fresh investigation — sharing your current Gateway build number and a short repro would help us a lot.

Thanks again for taking the time to write this up. 🙏

— The Windows Admin Center team
